doc/user/sql/create-sink.md

```diff
@@ -82,6 +80,43 @@ FORMAT AVRO USING
     CONFLUENT SCHEMA REGISTRY 'http://localhost:8081';
 ```
 
+### Avro OCF sinks
```

Comment:
Add an example here and for the Kafka sink case showing how to look up actual file/topic name